cmd/config.go

```diff
@@ -0,0 +1,103 @@
+/*
+ * JuiceFS, Copyright (C) 2021 Juicedata, Inc.
+ *
+ * This program is free software: you can use, redistribute, and/or modify
+ * it under the terms of the GNU Affero General Public License, version 3
+ * or later ("AGPL"), as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package main
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/juicedata/juicefs/pkg/meta"
+	"github.com/urfave/cli/v2"
+)
+
+var changableConfigs = []cli.Flag{
+	&cli.Uint64Flag{
+		Name:  "capacity",
+		Value: 0,
+		Usage: "the limit for space in GiB",
+	},
+	&cli.Uint64Flag{
+		Name:  "inodes",
+		Value: 0,
+		Usage: "the limit for number of inodes",
+	},
+	&cli.StringFlag{
+		Name:  "access-key",
+		Usage: "Access key for object storage (env ACCESS_KEY)",
```

Comment:
Should we take the ACCESS_KEY from env?

User may not want to expose the secret in command line history